pkg/executor/adapter.go

```diff
@@ -461,6 +461,15 @@ func IsFastPlan(p base.Plan) bool {
 	return false
 }
 
+func syncInternalSystemVariable(sctx *variable.SessionVars) {
```

Comment:
Does `loadCommonGlobalVariablesIfNeeded` has any bug or is it not called in some internal session?